io/net/bonding.py

```diff
@@ -195,6 +194,7 @@ def ping_check(self):
         '''
         ping check
         '''
+        time.sleep(10)
```

Comment:
why is sleep needed here ?

Reply:
it need few sec before pinging otherwise its failing

Reply:
interface_wait() did not help ?

Reply:
no

Reply:
Can you please elaborate on why it did not work ?

Reply:
before ping test what ever operation we are doing it need some time to set.i tried with wait() also this is not working.

Reply:
@smruti77 can you document the reason for not working as a comment in the test, so that the reason for using sleep is documented.

Reply:
@narasimhan-v  in script ..?

Reply:
yes. add a comment